plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergWritableTableHandle.java

```diff
@@ -43,7 +45,8 @@ public IcebergWritableTableHandle(
             @JsonProperty("partitionSpecAsJson") String partitionSpecAsJson,
             @JsonProperty("inputColumns") List<IcebergColumnHandle> inputColumns,
             @JsonProperty("outputPath") String outputPath,
-            @JsonProperty("fileFormat") FileFormat fileFormat)
+            @JsonProperty("fileFormat") FileFormat fileFormat,
+            @JsonProperty("properties") Map<String, String> properties)
```

Comment:
nit: maybe classify with some adjective (e.g. `storageProperties`), so it is more obvious that we are not talking about Trino's TableProperties here.